lisa/platform_.py

```diff
@@ -221,6 +229,27 @@ def delete_environment(self, environment: Environment) -> None:
     def cleanup(self) -> None:
         self._cleanup()
 
+    def _initialize_guest_nodes(self, node: Node) -> None:
+        platform_runbook = cast(schema.Platform, self.runbook)
+
+        if not platform_runbook.guests:
+            raise LisaException(
+                "guests must be defined in the platform runbook, "
+                "when guest_enabled is True."
+            )
+
+        for guest_runbook in platform_runbook.guests:
+            guest_runbook = cast(schema.GuestNode, guest_runbook)
+            # follow parent capability, so it can pass requirements validations.
+            guest_runbook.capability = node.capability
+            guest_node = node.create(
+                index=len(node.guests),
+                runbook=guest_runbook,
+                logger_name=f"{node.log.name}g",
```

Comment:
what does the suffix 'g' mean?

Reply:
g means guest. The prefix needs to be shorter to print more information in the same line.